src/ResourceManager/Compute/Commands.Compute/Generated/VirtualMachineScaleSetVM/VirtualMachineScaleSetVMListMethod.cs

```diff
@@ -35,17 +35,65 @@ public partial class InvokeAzureComputeMethodCmdlet : ComputeAutomationBaseCmdle
         protected object CreateVirtualMachineScaleSetVMListDynamicParameters()
         {
             dynamicParameters = new RuntimeDefinedParameterDictionary();
-            var pParameters = new RuntimeDefinedParameter();
-            pParameters.Name = "VirtualMachineScaleSetVMListParameters";
-            pParameters.ParameterType = typeof(Microsoft.Azure.Management.Compute.Models.VirtualMachineScaleSetVMListParameters);
-            pParameters.Attributes.Add(new ParameterAttribute
+            var pExpandExpression = new RuntimeDefinedParameter();
+            pExpandExpression.Name = "ExpandExpression";
+            pExpandExpression.ParameterType = typeof(string);
+            pExpandExpression.Attributes.Add(new ParameterAttribute
             {
                 ParameterSetName = "InvokeByDynamicParameters",
                 Position = 1,
-                Mandatory = true
+                Mandatory = false
+            });
+            pExpandExpression.Attributes.Add(new AllowNullAttribute());
+            dynamicParameters.Add("ExpandExpression", pExpandExpression);
+
+            var pFilterExpression = new RuntimeDefinedParameter();
+            pFilterExpression.Name = "FilterExpression";
+            pFilterExpression.ParameterType = typeof(string);
+            pFilterExpression.Attributes.Add(new ParameterAttribute
+            {
+                ParameterSetName = "InvokeByDynamicParameters",
+                Position = 2,
+                Mandatory = false
+            });
+            pFilterExpression.Attributes.Add(new AllowNullAttribute());
+            dynamicParameters.Add("FilterExpression", pFilterExpression);
+
+            var pResourceGroupName = new RuntimeDefinedParameter();
+            pResourceGroupName.Name = "ResourceGroupName";
```

Comment:
In the MAML library, ResourceGroupName parameter is inside VirtualMachineScaleSetVMListParameters object.  Is it a spec bug?